deploy.go

```diff
@@ -343,15 +343,15 @@ func (p *deploycmd) updateAppConfig(appf *appfile) error {
 
 	_, err := p.Apps.PatchAppsApp(param)
 	if err != nil {
-		params := clientApps.NewPostAppsParams() //XXX switch to put when v2.0 Fn
-		param.Body = &models.AppWrapper{
+		postParams := clientApps.NewPostAppsParams() //XXX switch to put when v2.0 Fn
```

Comment:
I see that only var name changes, so, how this solves the problem? Am i missing something?

Reply:
The original bug is that it’s assigning the body to the function param not the new var, I renamed the var to dismbiguate ‘param’ and ‘params’

Reply:
okay, can we add a test for this?

Reply:
Not easily right now, will look at improving CLI tests afterwards